EntAppProject/src/main/resources/static/css/JobBookCSS.css

```diff
@@ -1,20 +1,20 @@
-h1	{
-	color: green;
-	text-align: center;	
-}
-
-h2	{
-	text-align: center;	
-}
-
-span	{
-	color: orange
-}
-
-form	{
-	text-align: center;
-}
-
-fieldset	{
-	margin: 0px auto;
+h1	{
+	color: green;
+	text-align: center;	
+}
+
+h2	{
+	text-align: center;	
+}
+
+span.page-title	{
```

Comment:
This is part of an overarching suggestion for the team. This project only has one CSS file so far. The problem with that is that the styles are too generic. For example, the style 
```css
span { color: orange; }
```
will make every `span` orange across the application. This has not caused any issues so far because the application only uses spans in one way - `h1` headers. However, spans are a universal markup that have many use cases. If later down the road the team decides to add another span, then they will have to continuously overwrite this style, which I consider to be poor maintenance. I recommend adding extra detail to a style (such as a class name) to make the purpose of the style clear and to reduce unintentionally issues.